department-of-veterans-affairs/va.gov-cms
# VACMS-14701 Update vba facility fields

## Description

Relates to #14701

## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As an admin
1. go to https://pr15143-x90pmpzpeuow6pi4uyluoagtj5rpzncy.ci.cms.va.gov/salt-lake-city-va-regional-benefit-office-at-george-e-wahlen-department-of-veterans-affairs-medical-center/locations/pre-discharge-site-at-fort
   - [x] Validate that you see DB data fields in the "Why can't I edit this gray box.
   - [x] Validate the order 
      - Title
      - operating status
      - page intro and photo (those are empty so not showing on node view)
      - Facility data 
      - Prepare for your visit (empty so not showing)

2. Click the edit tab
   - [x] Validate that the Non-VA location official name, Non-VA location URL, Parent office and Shared VHA location field appear in the Facility Data box.  
   - [x] validate that since you are admin you can actually edit most of the fields  (not the Name of facility and locator api id those are ultra locked)
   - [x] Validate the order 
      - Title
      - operating status
      - page intro and photo 
      - Facility data 
      - Prepare for your visit 
      - editorial workflow
3. Change the status and save the node as draft
   - [x] validate that the change of status happens.

Login as MicheleMid
5. Go to https://pr15143-x90pmpzpeuow6pi4uyluoagtj5rpzncy.ci.cms.va.gov/salt-lake-city-va-regional-benefit-office-at-george-e-wahlen-department-of-veterans-affairs-medical-center/locations/pre-discharge-site-at-fort
6. Click the edit tab  
- [x] Validate that you see all the facility api data and db data but can not edit it
- [x] validate that you can change the status and save the node successfully. 

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
